src/components/Library/MonthVoting.tsx

```diff
@@ -53,10 +53,12 @@ const MonthVoting = (props: voteAllInfoProps) => {
         {verticalScrollInfo.map((vote: VoteInfo, idx: number) =>
           idx === verticalScrollInfo.length - 1 ? (
             <div key={idx} ref={ref}>
-              <EndedVoting id={vote.id} voteData={vote} key={idx}></EndedVoting>
+              <EndedVoting key={idx} id={vote.id} voteData={vote}></EndedVoting>
             </div>
           ) : (
-            <EndedVoting id={vote.id} voteData={vote} key={idx} isStart={idx === 0 ? true : false}></EndedVoting>
+            <div key={idx}>
+              <EndedVoting id={vote.id} voteData={vote} key={idx} isStart={idx === 0 ? true : false}></EndedVoting>
```

Comment:
```suggestion
<EndedVoting id={vote.id} voteData={vote} key={idx} isStart={!idx}></EndedVoting>
```
이렇게 하면 굳이 비교안해도 델것가트데!